app/javascript/mastodon/features/ui/components/column_loading.js

```diff
@@ -3,17 +3,25 @@ import PropTypes from 'prop-types';
 
 import Column from '../../../components/column';
 import ColumnHeader from '../../../components/column_header';
+import ImmutablePureComponent from 'react-immutable-pure-component';
 
-const ColumnLoading = ({ title = '', icon = ' ' }) => (
-  <Column>
-    <ColumnHeader icon={icon} title={title} multiColumn={false} focusable={false} />
-    <div className='scrollable' />
-  </Column>
-);
+export default class ColumnLoading extends ImmutablePureComponent {
 
-ColumnLoading.propTypes = {
-  title: PropTypes.node,
-  icon: PropTypes.string,
-};
+  static propTypes = {
+    title: PropTypes.node,
+    icon: PropTypes.string,
+  }
 
-export default ColumnLoading;
+  render() {
+    let { title, icon } = this.props;
+    title = title || '';
+    icon = icon || '';
```

Comment:
```javascript
const { title = '', icon = '' } = this.props;
```

Is it not acceptable in the above writing style?

Reply:
Ah...

```javascript
static propTypes = {
  title: PropTypes.oneOfType(PropTypes.node, PropTypes.string),
  icon: PropTypes.string,
};

static defaultProps = {
  title: '',
  icon: '',
};
```

The above may be better.

Reply:
Done! :)